src/pages/[platform]/build-a-backend/data/aws-appsync-apollo-extensions/index.mdx

```diff
@@ -0,0 +1,398 @@
+
+</InlineFilter>
+
+## Features
+
+AWS AppSync Apollo Extensions provide AWS AppSync authorizers to be used with the Apollo client to make it simple to apply the correct authorization payloads to your GraphQL operations.
+
+The Amplify library provides components to facilitate configuring the authorizers with Apollo client by providing configuration values to connect to your Amplify Data backend.
+
+### Install the AWS AppSync Apollo Extensions library
+
+<InlineFilter filters={["android"]}>
+
+To connect Apollo to AppSync without using Amplify, add the `apollo-appsync` dependency to your app/build.gradle.kts file. If your
+application is using Amplify Android, add the `apollo-appsync-amplify` dependency instead.
+
+```kotlin title="app/build.gradle.kts"
+dependencies {
+    // highlight-start
+    // Connect Apollo to AppSync without using Amplify
+    implementation("com.amplifyframework:apollo-appsync:1.0.0")    
+    // highlight-end
+    // or
+    // highlight-start
+    // Connect Apollo to AppSync, delegating some implementation details to Amplify
+    implementation("com.amplifyframework:apollo-appsync-amplify:1.0.0")    
+    // highlight-end
+}
+```
+
+</InlineFilter>
+
+<InlineFilter filters={["swift"]}>
+
+Add AWS AppSync Apollo Extensions into your project using Swift Package Manager.
+
+Enter its GitHub URL (`https://github.com/aws-amplify/aws-appsync-apollo-extensions-swift`), select **Up to Next Major Version** and click **Add Package**
+
+* Select the following libraries:
+    * **AWSAppSyncApolloExtensions**
+
+</InlineFilter>
+
+### Connecting to AWS AppSync with Apollo client
+
+AWS AppSync supports the following [authorization modes](https://docs.aws.amazon.com/appsync/latest/devguide/security-authz.html):
+
+#### API_KEY
+
+<InlineFilter filters={["swift"]}>
+
+```swift
+import AWSAppSyncApolloExtensions
+
+let authorizer = APIKeyAuthorizer(apiKey: "[API_KEY]")
+let interceptor = AppSyncInterceptor(authorizer)
+```
+
+</InlineFilter>
+
+<InlineFilter filters={["android"]}>
+
+An `ApiKeyAuthorizer` can be used with a hardcoded API key, by fetching the key from some source, or reading it from `amplify_outputs.json`:
+
+```kotlin
+// highlight-start
+// Use a hard-coded API key
+val authorizer = ApiKeyAuthorizer("[API_KEY]")
+//highlight-end
+// or
+// highlight-start
+// Fetch the API key from some source. This function may be called many times,
+// so it should implement appropriate caching internally.
+val authorizer = ApiKeyAuthorizer { fetchApiKey() }
+//highlight-end
+// or
+// highlight-start
+// Using ApolloAmplifyConnector to read API key from amplify_outputs.json
+val connector = ApolloAmplifyConnector(context, AmplifyOutputs(R.raw.amplify_outputs))
+val authorizer = connector.apiKeyAuthorizer()
+//highlight-end
+```
+
+</InlineFilter>
+
+#### AMAZON_COGNITO_USER_POOLS
+
+<InlineFilter filters={["swift"]}>
+
+If you are using Amplify Auth, you can create a method that retrieves the Cognito access token
```

Comment:
Amplify auth

Reply:
We use capitalized category names elsewhere in our documentation (for example, [here](https://docs.amplify.aws/android/build-a-backend/auth/set-up-auth/) and [here](https://docs.amplify.aws/android/build-a-backend/data/set-up-data/))